codegen/src/generator/json.rs

```diff
@@ -40,6 +40,7 @@ impl GenerateProtocol for JsonGenerator {
                 output_type = output_type,
                 request_uri = operation.http.request_uri,
                 target_prefix = service.metadata.target_prefix.as_ref().unwrap(),
+                json_version = service.metadata.json_version.as_ref().map(|x| x as &str).unwrap_or("1.0")
```

Comment:
Yep. Since this is in rusoto_codegen, it's something that will happen in the build stage, and so we'll know before even committing a change that it doesn't work, so I think we should prefer failing loudly rather than potentially doing the wrong thing for stuff in rusoto_codgen in general.